src/backend/distributed/commands/dependencies.c

```diff
@@ -681,6 +707,22 @@ ShouldPropagateObject(const ObjectAddress *address)
 }
 
 
+bool
+ShouldPropagateObjects(List *addresses)
```

Comment:
As noted, lets update the function name to `ShouldPropagateAnyObject`


Also, all new functions need function comment